### DISCUSSION TITLE
Is the illa supports chart？

### DISCUSSION THREAD

**Discussion by dualseason** [illacloud/illa-builder#347] [Q&A]
I want to know if illa supports chart form and how I can create it

**Comment by dualseason** [comment]
![image](https://user-images.githubusercontent.com/58629523/189836884-e491052c-b97e-4738-8b50-35f40d9def34.png)
just like the picture

**Reply by cyx0928** [reply]
Hey There:

Thx you for checking out our product. We have updated many features in the last couple of months., especially in components right now. Would you plz check out our product now? You can find a chart component in our product now.

Best,

ILLA team

**Comment by AruSeito** [comment]
We will support the chart in the near future
